clients/vanilla-rest.md

```diff
@@ -30,6 +30,29 @@ For example, to find the records where field _status_ is not equal to **active**
 GET /messages?status[$ne]=active
 ```
 
+To get really complex queries using the URL syntax, the back-end must implement the extended query string qs() (query string parser) middleware layer.  This is accomplished within the code by adding a 
+```
+app.set('query parser', 'extended');
```

Comment:
Does it make any difference doing what you've written here or doing the following?

```
.use(bodyParser.urlencoded({ extended: true }))
```